### ISSUE TITLE
DB Conversion to Brackets / Legacy throws errors because of mysql_real_escape_string, better use $wpdb->prepare 

### ISSUE THREAD

**Issue by herrvigg** [herrvigg/qtranslate-x#37]
<a href="https://github.com/kuchenundkakao"><img src="https://avatars1.githubusercontent.com/u/3602724?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [kuchenundkakao](https://github.com/kuchenundkakao)**
_Tuesday Feb 24, 2015 at 12:11 GMT_
_Originally opened as https://github.com/qTranslate-Team/qtranslate-x/issues/37_

----

As the title says, if you try to use the "convert database" function, you get a lot of errors because no mysql connection is opened.

Please replace lines 81 and 97 in the admin/admin_utils.php with the following:

```
$wpdb->query($wpdb->prepare('UPDATE '.$wpdb->posts.' set post_content = %s, post_title=%s, post_excerpt=%s  WHERE ID=%d',$content, $title, $excerpt, $post->ID));
```

http://codex.wordpress.org/Data_Validation#Database

**Comment by herrvigg** [comment]
<a href="https://github.com/mweimerskirch"><img src="https://avatars3.githubusercontent.com/u/362092?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mweimerskirch](https://github.com/mweimerskirch)**
_Wednesday Feb 25, 2015 at 23:22 GMT_

----

@kuchenundkakao you can edit the file right here on GitHub by clicking on the small pencil in the top right corner of the file and follow the messages on screen to create a "pull request". This way, your fix will get integrated much faster.

**Comment by herrvigg** [comment]
<a href="https://github.com/kuchenundkakao"><img src="https://avatars1.githubusercontent.com/u/3602724?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [kuchenundkakao](https://github.com/kuchenundkakao)**
_Thursday Feb 26, 2015 at 11:52 GMT_

----

Thanks, done that ;)

**Comment by herrvigg** [comment]
<a href="https://github.com/mweimerskirch"><img src="https://avatars3.githubusercontent.com/u/362092?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mweimerskirch](https://github.com/mweimerskirch)**
_Friday Feb 27, 2015 at 08:17 GMT_

----

This can be closed. Fixed by #48
